myFitnessTrainer/src/components/GenerateNewPlanAlgoButton.js

```diff
@@ -112,15 +99,15 @@ const GenerateNewPlanAlgoButton = ({
         };
         await postDocument("workout_plans", postObject);
 
-        navigation.replace("Dashboard");
+        navigation.goBack("Dashboard");
     };
 
     return (
         <TouchableOpacity
             style={styles.button}
             onPress={handleCreateNewPlanAlgo}
         >
-            <Text style={styles.buttonText}>Create Workout Plan</Text>
+            <Text style={styles.buttonText}>Generate Fitness Plan</Text>
```

Comment:
I had updated this text to be 'Create Workout Plan', so we are consistent in referencing the plan as a 'workout plan' vs. 'fitness plan'. I'm good with changing it to 'Generate Workout Plan'. Thoughts?

Reply:
Sorry about the confusion. 'Create Workout Plan' sounds good.